serverless/lambda/src/main/java/uk/gov/di/helpers/ApiGatewayResponseHelper.java

```diff
@@ -3,13 +3,29 @@
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.di.entity.ErrorResponse;
 
 public class ApiGatewayResponseHelper {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(ApiGatewayResponseHelper.class);
+
     public static <T> APIGatewayProxyResponseEvent generateApiGatewayProxyResponse(
             int statusCode, T body) throws JsonProcessingException {
-        ObjectMapper objectMapper = new ObjectMapper();
-        return generateApiGatewayProxyResponse(statusCode, objectMapper.writeValueAsString(body));
+        return generateApiGatewayProxyResponse(
+                statusCode, new ObjectMapper().writeValueAsString(body));
+    }
+
+    public static <T> APIGatewayProxyResponseEvent generateApiGatewayProxyErrorResponse(
+            int statusCode, ErrorResponse errorResponse) {
+        try {
+            return generateApiGatewayProxyResponse(
+                    statusCode, new ObjectMapper().writeValueAsString(errorResponse));
+        } catch (JsonProcessingException e) {
+            LOGGER.warn("Unable to generateApiGatewayProxyErrorResponse: " + e);
+            return generateApiGatewayProxyResponse(500, "");
```

Comment:
do we want to add something to the empty string in the request or can you just pass in null?